CMakeModules/FindCheck.cmake

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
why do you need this file?
Is it not enough to only use `pkg_check_modules(CHECK REQUIRED check)`?

Reply:
Indeed, I don't need it, thx!
I later discovered pkg_check_modules(), and forgot to remove this :-)